app/view/twig/editcontent/fields/_image.twig

```diff
@@ -56,11 +56,13 @@
 {% for attrib in option.attrib|default([]) %}
     {% set aid = attrib|lower|replace({'alt text': 'alt'}) %}
 
-    {% if aid in ['title', 'alt'] %}
+    {% if aid is not empty %}
         {% if aid == 'title' %}
             {% set atitle = __('general.phrase.title') %}
         {% elseif aid == 'alt' %}
             {% set atitle = __('field.image.label.alt') %}
+        {% else %}
+            {% set atitle = aid %}
```

Comment:
{% set atitle = attrib %}

Reply:
that would make that special 'alt text' case even weirder?

Reply:
My preference would be to delete the whole special 'alt text' case, but I have no clue why that was added there in the first case. It's not needed anymore now.

Reply:
I think the special 'alt text' is here for historical reasons. It is 3 years old and not documented so I think this replace can be removed too.